.travis.yml

```diff
@@ -13,18 +13,15 @@ env:
   matrix:
     - PGVERSION=9.5
     - PGVERSION=9.6
+    - PGVERSION=10
 before_install:
-  - git clone -b v0.6.1 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b pg10_hack --depth 1 https://github.com/citusdata/tools.git
```

Comment:
So this needs to be properly merged first?   What's the schedule there? Any issues?